### ISSUE TITLE
Alternative Text in README is Inaccessible

### ISSUE THREAD

**Issue by Tales-of-T** [OfficeDev/script-lab#573]
## Expected Behavior
The image descriptions should be useful to a user with a screen reader.

## Current Behavior
The text does not convey the same information as the video or images.

## Possible Solution
Make better alt text. 

## Steps to Reproduce (for bugs)
1. Open README.md
2. Open Narrator.
3. Have Narrator read the page.

## Context
Accessible documentation. 

## Your Environment
Edge.

**Comment by Zlatkovsky** [comment]
Thanks Toby!  Merged in now.  FYI, for small fixes like this, feel free to submit a pull request even without a bug -- no need to do extra overhead unless you want to...
